src/python/pants/backend/project_info/tasks/dependencies.py

```diff
@@ -38,6 +45,19 @@ def __init__(self, *args, **kwargs):
       raise TaskError('At most one of --internal-only or --external-only can be selected.')
 
   def console_output(self, unused_method_argument):
+    deprecated_conditional(
+      lambda: not self.is_external_only and not self.is_internal_only,
+      removal_version="1.26.0.dev1",
+      entity_description="External dependencies will no longer be included in dependencies output.",
```

Comment:
This results in the line:

> DeprecationWarning: DEPRECATED: External dependencies will no longer be included in dependencies output. will be removed in version 1.26.0.dev1.

Instead, you can rename this to be:

> The default dependencies output including external dependencies

So that it reads:

> DeprecationWarning: DEPRECATED: The default dependencies output including external dependencies will be removed in version 1.26.0.dev1.